api/views/CloudRecordSummaryView.py

```diff
@@ -256,10 +244,23 @@ def _token_to_id(self, token):
         except (urllib2.HTTPError,
                 urllib2.URLError,
                 httplib.HTTPException,
-                KeyError) as e:
-            raise e
-
+                KeyError) as error:
+            logger.error("%s: %s", type(error), str(error))
+            return None
+        logger.info("Token identifed as %s...", client_id[:15])
+        logger.debug("Full CLIENT_ID: %s", client_id)
         return client_id
 
     def _is_client_authorized(self, client_id):
-        return client_id in settings.ALLOWED_FOR_GET
+        """
+        Return true iff client_id can access summaries.
```

Comment:
I wondered if it might be, but it's not a common abbreviation in the code so it should probably be the whole phrase.